panel/src/mixins/forms/counter.js

```diff
@@ -0,0 +1,22 @@
+
+export default {
+  props: {
+    counter: {
+      type: Boolean,
+      default: true
+    }
+  },
+  computed: {
+    counterOptions() {
+      if (this.value === null || this.disabled || this.counter === false) {
+        return false;
+      }
+
+      return {
+        count: this.value && Array.isArray(this.value) ? this.value.length : 0,
```

Comment:
Value is `string` for `password`, `text` and `textarea` fields. 
I think there should be `string` or `array` control here. 
Am I wrong?

Reply:
Good catch! Added a fix to check for array/string.